client/js/app/components/common/react_multi_select.js

```diff
@@ -0,0 +1,98 @@
+var React = require('react');
```

Comment:
What was the rationale for building this yourself instead of using something like `ReactSelect` (http://jedwatson.github.io/react-select/)? This component is well built btw, but being custom is means more code (including css) to maintain ourselves over time.